### PR TITLE
 doc: Update the readme to BSP 1.2.2 which with A4 carrier board

### PR DESCRIPTION
**Pull request by JH989876525**
doc: Update the readme to BSP 1.2.2 which with A4 carrier board

### CODE REVIEW

File: tocs/2.Software/VVAS.md
Comment (HuiJu1218) [inline tocs/2.Software/VVAS.md:1]: The content of VVAS and VVAS DEMO have same content, perhaps users can refer to the VVAS page to check version.

File: tocs/3.POC/VVAS-Demo.md
Comment (HuiJu1218) [inline tocs/3.POC/VVAS-Demo.md:1]: - Please check out [this page](https://github.com/InnoIPA/EXMU-X261-usermanual/blob/main/tocs/2.Software/VVAS.md). ->For version and downloading VVAS, please refer to the xx page?
- At the beginning, remind users of the things they need to attention.
 --  PL APP 
 -- vvas plugin
 -- vvas gst 
-  screenshot of the installation RPM? How to check install sucess?

File: tocs/2.Software/utilities-intro.md
Comment (HuiJu1218) [inline tocs/2.Software/utilities-intro.md:1]: - Help -> FAQ (refer OTA page format)
- Cmd format
  -- Usage: cmd <<"value"> > (display issue, the value does not require quotation marks.)
  -- Example: cmd value

File: tocs/3.POC/VVAS-Demo.md
Comment (JH989876525) [reply tocs/3.POC/VVAS-Demo.md:1]: Fixed in commit bd8dda355c0f673e45393aa17f0425e484552a2a.

File: tocs/2.Software/utilities-intro.md
Comment (JH989876525) [reply tocs/2.Software/utilities-intro.md:1]: Fixed in commit bd5c3315f375e0c53578d1ee6dbd044d03df42dc.

File: tocs/2.Software/VVAS.md
Comment (JH989876525) [reply tocs/2.Software/VVAS.md:1]: Fixed in commit 4ef1322f2535f8ee10937f1a838aef65fc6ea674.

**Pull request merged**